README.md

```diff
@@ -29,6 +29,11 @@ Tracks the commits in a [git](http://git-scm.com/) repository.
 * `skip_ssl_verification`: *Optional.* Skips git ssl verification by exporting
   `GIT_SSL_NO_VERIFY=true`.
 
+* `tag_filter`: *Optional*. If specified the resource will only `trigger` when
```

Comment:
Description should instead say "resource will only detect commits with tags matching the filter" or some such (not trigger). I'll just fix that up after merging though.